src/main/java/jenkins/benchmark/jmh/BenchmarkFinder.java

```diff
@@ -1,40 +1,26 @@
 package jenkins.benchmark.jmh;
 
-import org.openjdk.jmh.runner.options.ChainedOptionsBuilder;
-import org.reflections.Reflections;
 
-import java.util.Objects;
-import java.util.Set;
+import net.java.sezpoz.Index;
+import net.java.sezpoz.IndexItem;
+import org.openjdk.jmh.runner.options.ChainedOptionsBuilder;
 
 /**
  * Find classes annotated with {@link JmhBenchmark} to run their benchmark methods.
  * @since 2.50
  */
+@SuppressWarnings("WeakerAccess")
 public final class BenchmarkFinder {
-    final private String[] packageName;
-
-    /**
-     * Creates a {@link BenchmarkFinder}
-     *
-     * @param packageNames find benchmarks in these packages
-     */
-    public BenchmarkFinder(String... packageNames) {
-        this.packageName = packageNames;
+    private BenchmarkFinder() {
     }
-
     /**
      * Includes classes annotated with {@link JmhBenchmark} as candidates for JMH benchmarks.
      *
      * @param optionsBuilder the optionsBuilder used to build the benchmarks
      */
-    public void findBenchmarks(ChainedOptionsBuilder optionsBuilder) {
-        Reflections reflections = new Reflections((Object[]) packageName);
-        Set<Class<?>> benchmarkClasses = reflections.getTypesAnnotatedWith(JmhBenchmark.class);
-        benchmarkClasses.forEach(clazz -> {
-            JmhBenchmark annotation = clazz.getAnnotation(JmhBenchmark.class);
-            if (Objects.nonNull(annotation)) {
-                optionsBuilder.include(clazz.getName() + annotation.value());
-            }
-        });
+    public static void findBenchmarks(ChainedOptionsBuilder optionsBuilder) {
+        for (IndexItem<JmhBenchmark, Object> item : Index.load(JmhBenchmark.class, Object.class)) {
+            optionsBuilder.include(item.className() + item.annotation().value());
```

Comment:
If you have to call `IndexItem.className` you are not using SezPoz correctly. If you do not care about lazy loading and just want to enumerate classes marked with a certain annotation, then use `org.jenkins-ci:annotation-indexer` (also already in the classpath).